internal/pkg/otel/README.md

```diff
@@ -35,69 +35,69 @@ This section provides a summary of components included in the Elastic Distributi
-| [jmxreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/jmxreceiver/v0.119.0/receiver/jmxreceiver/README.md) | v0.119.0 |
-| [k8sclusterreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/k8sclusterreceiver/v0.119.0/receiver/k8sclusterreceiver/README.md) | v0.119.0 |
-| [k8sobjectsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/k8sobjectsreceiver/v0.119.0/receiver/k8sobjectsreceiver/README.md) | v0.119.0 |
-| [kafkareceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/kafkareceiver/v0.119.0/receiver/kafkareceiver/README.md) | v0.119.0 |
-| [kubeletstatsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/kubeletstatsreceiver/v0.119.0/receiver/kubeletstatsreceiver/README.md) | v0.119.0 |
-| [nginxreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/nginxreceiver/v0.119.0/receiver/nginxreceiver/README.md) | v0.119.0 |
+| [filelogreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/filelogreceiver/v0.120.1/receiver/filelogreceiver/README.md) | v0.120.1 |
+| [hostmetricsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/hostmetricsreceiver/v0.120.1/receiver/hostmetricsreceiver/README.md) | v0.120.1 |
+| [httpcheckreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/httpcheckreceiver/v0.120.1/receiver/httpcheckreceiver/README.md) | v0.120.1 |
+| [jaegerreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/jaegerreceiver/v0.120.1/receiver/jaegerreceiver/README.md) | v0.120.1 |
+| [jmxreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/jmxreceiver/v0.120.1/receiver/jmxreceiver/README.md) | v0.120.1 |
+| [k8sclusterreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/k8sclusterreceiver/v0.120.1/receiver/k8sclusterreceiver/README.md) | v0.120.1 |
+| [k8sobjectsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/k8sobjectsreceiver/v0.120.1/receiver/k8sobjectsreceiver/README.md) | v0.120.1 |
+| [kafkareceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/kafkareceiver/v0.120.1/receiver/kafkareceiver/README.md) | v0.120.1 |
+| [kubeletstatsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/kubeletstatsreceiver/v0.120.1/receiver/kubeletstatsreceiver/README.md) | v0.120.1 |
+| [nginxreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/nginxreceiver/v0.120.1/receiver/nginxreceiver/README.md) | v0.120.1 |
 | [nopreceiver](https://github.com/open-telemetry/opentelemetry-collector/blob/receiver/nopreceiver/v0.119.0/receiver/nopreceiver/README.md) | v0.119.0 |
```

Comment:
Why are some components not bumped?

Reply:
I've bumped them in https://github.com/elastic/elastic-agent/pull/7050/commits/9baaec2d9d7ffa93f2d1259cdeebd790a365b751.